modules/mono/editor/script_templates/CharacterBody3D/basic_movement.cs

```diff
@@ -5,17 +5,28 @@
 
 public partial class _CLASS_ : _BASE_
 {
-    public const float Speed = 5.0f;
+    public const float Accel = 25.0f;
+    public const float Decay = 5.0f;
+    public const float Speed = Accel/Decay;
     public const float JumpVelocity = 4.5f;
 
     public override void _PhysicsProcess(double delta)
     {
+     // Vector3 position = Position;
         Vector3 velocity = Velocity;
+        float dt = (float)delta;
+        float f0 = 1f;
+        float f1 = dt;
+        float f2 = dt*dt/2f;
+        if (Decay>0f)
+            f0 = Mathf.exp(-dt*Decay);
+            f1 = ( 1f - f0 ) / Decay;
+            f2 = ( dt - f1 ) / Decay;
```

Comment:
I don't know if this PR is still relevant but since this is C# and not gdscript I think you want to add `{ }` (or make it unconditional)